Author: Javi Merino <vicho@debian.org>
Description: Ignore the return code of chmod in test-inherit-mode
 In freebsd, a newly created directory has the same group as the parent
 directory by default.  That means that the test directory created by
 test-inherit-mode.t is owned by root's group, so "chmod g+s .hg/store"
 fails to set the SGID bit and returns 1.  If we ignore chmod's return
 code, the testsuite passes again.
Forwarded: http://selenic.com/pipermail/mercurial-devel/2012-March/038464.html

--- a/tests/test-inherit-mode.t
+++ b/tests/test-inherit-mode.t
@@ -138,7 +138,7 @@ just check that directories have the sam
   $ hg init setgid
   $ cd setgid
   $ chmod g+rwx .hg/store
-  $ chmod g+s .hg/store 2> /dev/null
+  $ chmod g+s .hg/store 2> /dev/null || /bin/true
   $ mkdir dir
   $ touch dir/file
   $ hg ci -qAm 'add dir/file'
